### PR TITLE
Update HCLAppScanIASTReader.java

### PR DESCRIPTION
**Pull request by zivRhcl**

### CODE REVIEW

File: plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/HCLAppScanIASTReader.java [hunk 1]
```diff
@@ -114,33 +143,12 @@ private void parseFindings(TestSuiteResults tr, String json) throws Exception {
     }
 
     private int cweLookup(String rule) {
-        switch (rule) {
```
Comment (darkspirit510) [inline plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/HCLAppScanIASTReader.java:117]: Some of the cases are not present in your new map. For backwards compatibility, it should at least contain all old entries.

**Review by darkspirit510 — CHANGES_REQUESTED** [review]

File: plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/HCLAppScanIASTReader.java (hunk 1 above)
Comment (zivRhcl) [reply plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/HCLAppScanIASTReader.java:117]: Added the relevant cases to the map

Comment (davewichers) [reply plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/HCLAppScanIASTReader.java:117]: @darkspirit510 - You OK with this now so I should merge?

Comment (zivRhcl) [reply plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/HCLAppScanIASTReader.java:117]: @davewichers Can you merge it please?

**Pull request merged**